modules/persistent-storage-cinder-creating-pv.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-creating-pv_{context}"]
+= Creating the Persistent Volume
+
+You must define your persistent volume (PV) in an object definition before creating
+it in {product-title}:
+
+.Procedure
+
+. Save your object definition to a file.
++
+.azure-advanced-disk-storageclass.yaml
```

Comment:
File name is started with "azure"

Reply:
@chao007 Can I get an ack please that this PR looks good from QE?

Reply:
Yes. Thanks for update

Reply:
It looks like we're defining a PersistentVolume here. I'd update the header of this block to be `.azure-advanced-disk-persistentvolume.yaml`.